api/nodes/serializers.py

```diff
@@ -150,13 +151,17 @@ def create(self, validated_data):
         auth = Auth(user)
         node = self.context['view'].get_node()
         pointer_node = Node.load(validated_data['node']['_id'])
-        pointer = node.add_pointer(pointer_node, auth, save=True)
-        return pointer
+        if not pointer_node:
+            raise exceptions.NotFound('Node not found.')
```

Comment:
ParseError. Heh. Probably a custom handler for generic BadRequest would be fine. Does it actually return the phrase ParseError in the error response, or is that just a code thing? If it's just code, I think leaving it the standard handler is fine.